client/src/components/DataList/index.js

```diff
@@ -1,20 +1,20 @@
 import React from "react";
-import { uniq } from "lodash";
 
-class Datalist extends React.PureComponent {
+class DataList extends React.PureComponent {
   render() {
     let options = [];
-
     if (this.props.options && this.props.processOptions) {
       options = this.props.processOptions(this.props.options);
     } else if (this.props.options) {
       options = this.props.options;
     }
 
+    const uniqueOptions = [...new Set(options)];
+
     return (
       <datalist id={this.props.id}>
-        {uniq(options).map(option => (
-          <option key={option} value={option}>
+        {uniqueOptions.map(option => (
```

Comment:
I'm still confused about why we're changing this component.  In what situation would you be passing falsey or duplicate options and why?  The responsibility of handing this component the right data is on the caller, not on `Datalist`.

Reply:
After talking it through, I agree that it can be considered the within the realm of Datalist's responsibilities to ensure there are no duplicate options passed.